gstreamer-pbutils/src/lib.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2017 Sebastian Dröge <sebastian@centricular.com>
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#[macro_use]
+extern crate bitflags;
+extern crate libc;
+
+#[macro_use]
+extern crate glib;
+extern crate glib_sys as glib_ffi;
+extern crate gobject_sys as gobject_ffi;
+extern crate gstreamer as gst;
+extern crate gstreamer_sys as gst_ffi;
+extern crate gstreamer_pbutils_sys as ffi;
+
+macro_rules! callback_guard {
+    () => (
+        let _guard = ::glib::CallbackGuard::new();
+    )
+}
+
+macro_rules! assert_initialized_main_thread {
+    () => (
+        if unsafe {::gst_ffi::gst_is_initialized()} != ::glib_ffi::GTRUE {
```

Comment:
We should call `gst_pb_utils_init` here also, but there's nothing to check if it was already called before...

Reply:
OTOH it does not actually do much so we might as well just always call it here

Reply:
Related to this: https://bugzilla.gnome.org/show_bug.cgi?id=793611

Reply:
IMHO we should just call it here every time via a https://doc.rust-lang.org/std/sync/struct.Once.html

Reply:
It doesn't hurt but I thought the plan was to deprecate the `gst_pb_utils_init()` and just have one for the specific parts that need it. Anyway, added the Once there.

Reply:
Yes but then the pbutils bindings would only work with 1.14. At least for the next 2 years I'd like to depend on a lower version than 1.14 (currently 1.8).